.github/workflows/helm-integ-test.yml

```diff
@@ -0,0 +1,98 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+name: Run Helm Charts Integration Tests
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+    inputs:
+      cluster_name:
+        required: true
+        type: string
+        default: "cwagent-operator-helm-integ"
```

Comment:
Any specific reason to pass the cluster name instead of creating one as part integ test?

Reply:
This  default cluster might be deleted by someone in our test account and this integ test will fail in that case.

Reply:
It's just a prefix to a cluster name which should help with easier debugging. It's just another way not hard coding it in the terraform resource block.

Reply:
For me its confusing as it says the cluster name and we are passing it to test. Can we just maintain similar to other integ tests where we just include as part of terraform?